decidim/decidim
# Add Procfile support

#### :tophat: What? Why?
This PR is adding the Procfile.dev file and foreman to ease up the development workflow. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- References #10389
- Fixes #10518 

#### Testing
1. Checkout the branch
2. run `rake decidim:upgrade` task (to install package.json dependencies)
3. run `rake decidim:procfile:install`
4. run bin/dev task
5. see the decidim instance is up and running

:hearts: Thank you!
